src/main/java/uk/gov/hmcts/reform/locationrefdata/camel/mapper/ServiceToCcdServiceMapper.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.locationrefdata.camel.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.data.ingestion.camel.mapper.IMapper;
+import uk.gov.hmcts.reform.locationrefdata.camel.binder.ServiceToCcdService;
+
+import java.util.HashMap;
```

Comment:
are we following different rule set for LRD? why it  is not failing if imports are not in the proper order?

Reply:
It is default ruleset comes as part of HMCTS..rather we have to change rd which is big work. We are not following right one in rd...Same as above.